content/run.md

```diff
@@ -161,12 +161,12 @@ Set the environment variable for the directory as below:
 
 ```{discussion} Choose the global surface data file corresponding to the resolution! 
 For example: for 0.5x0.5 resolution, choose `surfdata_360x720cru_16pfts_Irrig_CMIP6_simyr2000_c170824.nc` (under shared noresm folder)
-and add this file location in the python script `subset_data.py`
+and add this file location in the python script `subset_data.py`. Make sure you have created a conda environment for python including xarray, netcdf4 installed or loaded on FRAM or SAGA.
```

Comment:
This doesn't belong here! Should go to line 167.

Reply:
Okay that's a good point. I moved the text there.